### ISSUE TITLE
Can't login to Google

### ISSUE THREAD

**Issue by jeid64** [rckclmbr/pyportify#28]
Hey, when I try to POST my Google Account username and password I get this stacktrace

2015-05-26 15:33:19,900 - pyportify.views - ERROR - Exception on /google/login [POST]
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Library/Python/2.7/site-packages/pyportify-0.2.4-py2.7.egg/pyportify/views.py", line 88, in google_login
    user_scope.google_login(email, password)
  File "/Library/Python/2.7/site-packages/pyportify-0.2.4-py2.7.egg/pyportify/views.py", line 60, in google_login
    self._google_loggedin = self.googleapi.login(username, password)
  File "/Library/Python/2.7/site-packages/pyportify-0.2.4-py2.7.egg/pyportify/google.py", line 55, in login
    auth = parse_auth_response(data).get("Auth", None)
  File "/Library/Python/2.7/site-packages/pyportify-0.2.4-py2.7.egg/pyportify/google.py", line 170, in parse_auth_response
    k, v = line.split("=", 1)
ValueError: need more than 1 value to unpack
127.0.0.1 - - [2015-05-26 15:33:19] "POST /google/login HTTP/1.1" 500 412 0.438640

This look familiar at all? I can try and use the command line app instead of the web browser one if interested.

**Comment by jeid64** [comment]
Actually, the command line app is the same thing.

Traceback (most recent call last):
  File "/usr/local/bin/pyportify-copyall", line 9, in <module>
    load_entry_point('pyportify==0.2.4', 'console_scripts', 'pyportify-copyall')()
  File "/Library/Python/2.7/site-packages/pyportify-0.2.4-py2.7.egg/pyportify/copy_all.py", line 17, in main
    app.user_scope.google_login(google_email, google_pass)
  File "/Library/Python/2.7/site-packages/pyportify-0.2.4-py2.7.egg/pyportify/views.py", line 60, in google_login
    self._google_loggedin = self.googleapi.login(username, password)
  File "/Library/Python/2.7/site-packages/pyportify-0.2.4-py2.7.egg/pyportify/google.py", line 55, in login
    auth = parse_auth_response(data).get("Auth", None)
  File "/Library/Python/2.7/site-packages/pyportify-0.2.4-py2.7.egg/pyportify/google.py", line 170, in parse_auth_response
    k, v = line.split("=", 1)
ValueError: need more than 1 value to unpack

**Comment by jeid64** [comment]
I modified google.py to print the line before it's split.

The line is "https://developers.google.com/identity/protocols/AuthForInstalledApps"

Curiously enough, Google sends a link telling you that ClientLogin is deprecated and is no longer available.

**Comment by jeid64** [comment]
ClientLogin will no longer work. I went ahead and tested a possible solution. I reverted to https://github.com/rckclmbr/pyportify/commit/4e631e9fe17e57791c761463e0dc9b6834814ee6 and went back to using the Unofficial Google Music API.

I used https://github.com/simon-weber/Unofficial-Google-Music-API/tree/develop as my gmusicapi library version to incorporate Simon's changes to the API to use OAuth instead of ClientLogin. This branch is super alpha-y (with Simon only getting OAuth working like 15 days ago) but pyportify now works with one minor change. I think it might be a good idea to switch back to the Google Music API instead of duplicating work and maintaining a complete OAuth implementation. 

My minor change to get pyportify working with the develop branch of the gmusicapi library can be found here (https://github.com/jeid64/pyportify/tree/gmusic_api_revert)
